### ISSUE TITLE
Replay last TTP/Chain button

### ISSUE THREAD

**Issue by khyberspache** [preludeorg/operator-support#549]
Having a "replay" last scheduled TTP/Chain would be very useful.

**Comment by privateducky** [comment]
Agreed. This would be a nice core feature, that would be ideal for now and through our 2.0 release.
